critiquebrainz/frontend/templates/macros.html

```diff
@@ -92,3 +92,11 @@
     {% endif %}
   </div>
 {%- endmacro %}
+
+{% macro soundcloud_player(soundcloud_url) %}
+  <div class=soundcloud-player">
+    {% if soundcloud_url %}
+      <iframe style="margin-top: 10px;margin-left: -5px;" width="100%" height="300" allowtransparency="true" scrolling="no" frameborder="no" src="https://w.soundcloud.com/player/?url={{soundcloud_url}}&auto_play=false"></iframe>
```

Comment:
it doesn't seem to working from the css file for some reason. I saw more iframes and div have it like this so I thought it was okay...